packages/core/src/mixins/__tests__/closable.test.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount } from 'enzyme';
+import keycode from 'keycode';
+
+import closable from '../closable';
+
+const Foo = () => (
+    <div id="foo">Foo</div>
+);
+
+jest.useFakeTimers();
+
+it('renders without crashing', () => {
```

Comment:
We forgot to use `describe()` to wrap all the test case in mixins. Orz

Reply:
I thinks it's fine not to use `describe()` inside mixins, since we're testing on the same target?
Unlike those “wrapped components”